core/controllers/skill_editor.py

```diff
@@ -146,6 +146,70 @@ class EditableSkillDataHandler(base.BaseHandler):
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'skill_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': constants.ENTITY_ID_REGEX
+                }]
+            }
+        }
+    }
+
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'PUT': {
+            'version': {
+                'schema': {
+                    'type': 'int',
+                }
+            },
+            'commit_message': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
```

Comment:
Is frontend sending `None` or empty string?

Reply:
Judging on the basis of the written handler, I'd say that it was sending None.

Reply:
Please check the frontend code.

Reply:
The data type of commit_message is string and I believe it is sending an empty string afaik. Please correct me if I am wrong.

Reply:
So should I make the default value ""?

Reply:
Can you link the frontend code so that I can take a look?